CONTRIBUTING.md

```diff
@@ -27,6 +27,15 @@ It's not you, it's us! _fastlane_ and its related tools receive a lot of issues
 
 One way we do this is by closing issues that we don't feel are immediately actionable. This might mean that we need more information in order to investigate. Or, it might mean that we haven't been able to reproduce it using the provided info. In this case we might close the issue while we wait for others to reproduce the problem and possibly provide some more info that unlocks the mystery.
 
+Another way we do this is by having an automated bot go through our issues and PRs. The main goal of the bot is to ensure that the issues are still relevant and reproducible. Issues can be opened, and later fall idle for a variety of reasons:
```

Comment:
It may be good to add a 

```
<a name="fastlane_bot"></a>
```
named anchor so that we could reference this section from, for exemple, the text the bot lets on issues